sailthru_import/test/test_sailthru_import.py

```diff
@@ -9,6 +9,7 @@
 @patch('sailthru.sailthru_client')
 class SailThruTestCase(unittest.TestCase):
     def test_upload_csv_to_sailthru(self, mock_client):
-        mock_client.return_value.api_get.response.body.status.return_value = "completed"
-        mock_client.upload_csv_to_sailthru('test.csv', 'list name', 'bfohl@edx.org', mock_client)
+        mock_client.api_get.return_value.get_body.return_value = {'status': 'completed'}
+        mock_client.api_post.return_value.is_ok.return_value = True
+        sailthru_import.upload_csv_to_sailthru('test.csv', 'list name', 'bfohl@edx.org', mock_client)
         mock_client.upload_csv_to_sailthru.assert_called_once()
```

Comment:
Since `upload_csv_to_sailthru` isn't a method on the sailthru client, I'm surprised if this passes.